### PR TITLE
Added a very basic description on stand-ups

### PR DESCRIPTION
**Pull request by davinderpalrehal**

### CODE REVIEW

File: src/pages/agile/daily-standup-and-daily-scrum/index.md [hunk 1]
```diff
@@ -3,13 +3,17 @@ title: Daily Standup and Daily Scrum
...
 
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/agile/daily-standup-and-daily-scrum/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
+Daily 10 min (or slightly more) meeting with the whole team just bringing up any major obstacles that are stopping progress. The team also talks about what they are going to be working on that day.
 
-<a href='https://github.com/freecodecamp/guides/blob/master/README.md' target='_blank' rel='nofollow'>This quick style guide will help ensure your pull request gets accepted</a>.
+
+<!--
+This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/agile/daily-standup-and-daily-scrum/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>. -->
+
+<!--<a href='https://github.com/freecodecamp/guides/blob/master/README.md' target='_blank' rel='nofollow'>This quick style guide will help ensure your pull request gets accepted</a>.-->
```
Comment (erictleung) [inline src/pages/agile/daily-standup-and-daily-scrum/index.md]: These comments can be removed as they are just guides on how to get started.

File: src/pages/agile/daily-standup-and-daily-scrum/index.md [hunk 2]
```diff
@@ -3,13 +3,17 @@ title: Daily Standup and Daily Scrum
...
 
 <!-- The article goes here, in GitHub-flavored Markdown. Feel free to add YouTube videos, images, and CodePen/JSBin embeds  -->
 
 #### More Information:
 <!-- Please add any articles you think might be helpful to read before writing the article -->
-
+<a href="https://en.wikipedia.org/wiki/Stand-up_meeting" target='_blank'>Wikipedia article</a>
```
Comment (erictleung) [inline src/pages/agile/daily-standup-and-daily-scrum/index.md]: Can you change anchor text from "Wikipedia article" to "Stand-up meeting"? Thanks!

**Comment by davinderpalrehal** [comment]
@erictleung made the changes please have a look

**Comment by IsaacAbrahamson** [comment]
LGTM :+1:

**Pull request merged**